src/isar/apis/models/start_mission_definition.py

```diff
@@ -9,29 +10,32 @@
 from robot_interface.models.mission.step import (
     STEPS,
     DriveToPose,
-    InspectionStep,
     TakeImage,
     TakeThermalImage,
     TakeThermalVideo,
     TakeVideo,
 )
 
-inspection_step_types: List[Type[InspectionStep]] = [
-    TakeImage,
-    TakeThermalImage,
-    TakeVideo,
-    TakeThermalVideo,
-]
+
+class InspectionTypes(str, Enum):
+    image = "Image"
+    thermal_image = "ThermalImage"
+    video = "Video"
+    thermal_video = "ThermalVideo"
+
+
+class StartMissionInspectionDefinition(BaseModel):
+    type: InspectionTypes = Field(default=InspectionTypes.image)
+    inspection_target: InputPosition
+    analysis_types: Optional[List]
+    duration: Optional[float]
+    metadata: Optional[dict]
 
 
 class StartMissionTaskDefinition(BaseModel):
     pose: InputPose
     tag: Optional[str]
-    inspection_target: InputPosition
-    inspection_types: List[str] = Field(
-        default=[step.get_inspection_type().__name__ for step in inspection_step_types]
-    )
-    video_duration: Optional[float] = Field(default=10)
+    inspections: List[StartMissionInspectionDefinition]
```

Comment:
Won't the inspection target always be the same for each inspection in a task? Because there is one tag per task?
Should it therefore be outside of the `StartMissionInspectionDefinition` ?

Reply:
Had a discussion with others on the team, but concluded that having the possibility to adjust target but standing on the same place might be beneficial, especially when inspecting a pipe or larger tags